interlok-core/src/test/java/com/adaptris/core/MultiPayloadAdaptrisMessageImpTest.java

```diff
@@ -62,5 +67,22 @@ public void testRemovePayloadMessageHeader() {
     assertFalse(multiPayloadAdaptrisMessage.payloadHeadersContainsKey("key"));
     assertFalse(multiPayloadAdaptrisMessage.payloadHeadersContainsKey("another-payload", "key"));
   }
+  
+  //Unit test added to test bug INTERLOK-4325 
+  @Test
+  public void testUpdatingOriginalPayloadFromInputStream() throws IOException {
+	System.out.println(PROPERTIES.getProperty(EXAMPLE_PAYLOAD_FILE));
+	MultiPayloadAdaptrisMessage multiPayloadAdaptrisMessage = MessageHelper.createMultiPayloadMessage("default-payload", PROPERTIES.getProperty(EXAMPLE_PAYLOAD_FILE));
+    System.out.println(multiPayloadAdaptrisMessage.getContent("default-payload"));
+    multiPayloadAdaptrisMessage.setContent("new payload", multiPayloadAdaptrisMessage.getContentEncoding());
+    
+
+    assertEquals("new payload", multiPayloadAdaptrisMessage.getContent());
+    
+    multiPayloadAdaptrisMessage.addContent("default-payload", "another payload", multiPayloadAdaptrisMessage.getContentEncoding());
+
+    assertEquals("another payload", multiPayloadAdaptrisMessage.getContent());
+
```

Comment:
Remove empty line

Reply:
removed unneeded empty lines and just left the 1 empty line between the two assert equals for ease of reading(but can remove this line if better)